### PR TITLE
first pass at coursier doc

### PR DESCRIPTION
**Pull request by JosephMoniz**
A first pass at adding some simple coursier documentation.

### CODE REVIEW

File: index.mako [hunk 1]
```diff
@@ -576,8 +576,17 @@ It has been tested on linux, macOS, and windows.
...
+[application descriptors](https://get-coursier.io/docs/2.0.0-RC6-10/cli-install.html#application-descriptor-reference)
+for the hook to install. For configuring coursier hooks, your
+[`entry`](#hooks-entry) should correspond to an executable installed from the
+repositories `.pre-commit-channel` folder.
+
 __Support:__ `coursier` hooks are known to work on any system which has the `cs`
-package manager installed.  It has been tested on linux.
+package manager installed. The specific coursier applications you install may depend
+on various versions of the JVM, consult the hooks documentation for clarificaiton.
+This integration has been tested on linus
```
Comment (asottile) [inline index.mako:589]:
```suggestion
on various versions of the JVM, consult the hooks' documentation for clarification.
It has been tested on linux.
```

File: index.mako [hunk 2]
```diff
@@ -576,8 +576,17 @@ It has been tested on linux, macOS, and windows.
 
 _new in 2.8.0_
 
+The hook repository muist have a `.pre-commit-channel` folder and that folder must contain
+the coursier
+[application descriptors](https://get-coursier.io/docs/2.0.0-RC6-10/cli-install.html#application-descriptor-reference)
+for the hook to install. For configuring coursier hooks, your
+[`entry`](#hooks-entry) should correspond to an executable installed from the
+repositories `.pre-commit-channel` folder.
```
Comment (asottile) [inline index.mako:584]:
```suggestion
repository's `.pre-commit-channel` folder.
```

File: index.mako [hunk 3]
```diff
@@ -576,8 +576,17 @@ It has been tested on linux, macOS, and windows.
 
 _new in 2.8.0_
 
+The hook repository muist have a `.pre-commit-channel` folder and that folder must contain
```
Comment (asottile) [inline index.mako:579]:
```suggestion
The hook repository must have a `.pre-commit-channel` folder and that folder must contain
```

**Comment by JosephMoniz** [comment]
@asottile thanks for the review! I need to l2spell .... that or configure my spell checker.

**Review by asottile — APPROVED** [review]

**Pull request merged**